### PR TITLE
added administration section and updated existing jsp for genre and booking

### PR DESCRIPTION
**Pull request by xtrnkal**
important for others when creating .jsp

### CODE REVIEW

File: web/src/main/java/cz/muni/fi/pa165/mvc/controllers/AdministrationController.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import org.springframework.validation.ObjectError;
+import org.springframework.web.bind.WebDataBinder;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import javax.validation.Valid;
+import javax.inject.Inject;
+import java.util.List;
+
```
Comment (xtrnkal) [inline web/src/main/java/cz/muni/fi/pa165/mvc/controllers/AdministrationController.java:6]: unused imports

File: web/src/main/java/cz/muni/fi/pa165/mvc/controllers/AdministrationController.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	 * Shows a list of genres, for admins with the ability to add, delete or edit.
+	 * 
+	 * @param model data to display
+	 * @return JSP page name
+	 */
+	@RequestMapping(value = "/", method = RequestMethod.GET)
+	public String list(Model model) {
+		model.addAttribute("genres", genreFacade.getAllGenres());
+		return "administration";
+	}
```
Comment (xtrnkal) [inline web/src/main/java/cz/muni/fi/pa165/mvc/controllers/AdministrationController.java:24]: okey, this isnt nice, i dont really need these facades

File: web/src/main/java/cz/muni/fi/pa165/mvc/controllers/AdministrationController.java (hunk 1 above)
Comment (xtrnkal) [reply web/src/main/java/cz/muni/fi/pa165/mvc/controllers/AdministrationController.java:6]: fixed

File: web/src/main/java/cz/muni/fi/pa165/mvc/controllers/AdministrationController.java (hunk 2 above)
Comment (xtrnkal) [reply web/src/main/java/cz/muni/fi/pa165/mvc/controllers/AdministrationController.java:24]: fixed

File: web/src/main/webapp/WEB-INF/tags/administrationTemplate.tag [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@ taglib prefix="s" uri="http://www.springframework.org/tags" %>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form" %>
+<%@ attribute name="body_area" fragment="true" %>
+<%@ attribute name="subtitle" required="false" %>
+
+<my:pagetemplate title="Administration">
+    <jsp:attribute name="body">
+        <div class="btn-group btn-group-justified">
+            <a href="/web/show/" class="btn btn-primary">Shows</a>
```
Comment (rutor) [inline web/src/main/webapp/WEB-INF/tags/administrationTemplate.tag]: Dej tam `${pageContext.request.contextPath}`, at si muzeme byt jisti, ze se to nemuze rozbit

**Review by rutor — DISMISSED** [review]
Budou tam sice zbytecne merge konflikty (stacilo dat example, kdyz to ve vysledku jeste nic nedela), ale je to OK

File: web/src/main/webapp/WEB-INF/tags/administrationTemplate.tag (hunk 3 above)
Comment (xtrnkal) [reply web/src/main/webapp/WEB-INF/tags/administrationTemplate.tag]: pridano

**Review by rutor — APPROVED** [review]
OK

**Pull request merged**